src/main/resources/application.yaml

```
@@ -101,7 +101,7 @@ media-service: ${MEDIA_SERVICE:MediaKind}
 platform-env: ${PLATFORM_ENV_TAG:Staging}
 
 cron-user-email: ${CRON_USER_EMAIL:test@test.com}
-vodafone-user-email: ${VODAFONE_USER_EMAIL:vodafone@test.com}
+vodafone-user-email: ${VODAFONE_USER_EMAIL:dts-pre-app-stg@hmcts.net}
```

```suggestion
vodafone-user-email: ${CRON_USER_EMAIL:vodafone@test.com}
```
Might as well change to this, so it works in prod